API.md

```diff
@@ -400,6 +400,23 @@ Access: read only.
 Contains the default authentication configuration if a default strategy was set via
 [`server.auth.default()`](#server.auth.default()).
 
+#### <a name="server.caches" /> `server.caches`
+
+Access: read only.
+
+An object containing the cache provisions, where each key is a cache name and the values are objects with the following properties:
+
+- `segments` - an object where each key is a segment name and the values are [cache policies](https://hapi.dev/family/catbox/api#policy) created with [server.cache()](#server.cache()).
```

Comment:
Given that this is likely only ever going to be iterated over, I would prefer that it is exposed as a `Map`.

Actually, I would prefer an array with objects containing `segment` & `policy` properties. This would allow multiple entries for the same segment when sharing is enabled.

Reply:
And that array could just be the policy objects if we make the `_segment` property on the policy public (as `segment`).

Reply:
I like the idea of exposing the `_segment` property as `segment`. We are referring to [this property](https://github.com/hapijs/catbox/blob/60db7cd7fbed5521f0eb690630868f4e42609eb5/lib/policy.js#L78) right?